packages/form-participation/test-web/MixinWithValidators.lit.test.ts

```diff
@@ -0,0 +1,169 @@
+  });
+
+  it('will track the observedAttributes relative to validators', async () => {
+    expect(constructor.observedAttributes.includes('other')).to.be.true;
+  });
+
+  it('will return the validator from getValidator', async () => {
+    // @ts-ignore
+    expect(constructor.getValidator('required')).to.equal(requiredValidator);
+  });
+
+  it('will add validator attributes to observedAttributes', async () => {
+    expect(constructor.observedAttributes.includes('required')).to.be.true;
+  });
+
+  it('will mark the validity state as invalid given the callback', async () => {
+    expect(el.validity.valid).to.be.false;
+  });
+
+  it('will have a reference to the parent form', async () => {
+    expect(el.form).to.equal(form);
+  });
+
+  it('will not recommend showing an error when pristine', async () => {
+    expect(el.validity.valid).to.be.false;
+    expect(el.showError).to.be.false;
+  });
+
+  it('will recommend showing an error when touched/not focused', async () => {
+    expect(el.validity.valid).to.be.false;
+    el.dispatchEvent(new Event('focus'));
+    await elementUpdated(el as unknown as Element);
+    // expect(el.showError).to.be.false;
+    el.dispatchEvent(new Event('blur'));
+    await elementUpdated(el as unknown as Element);
+    expect(el.showError).to.be.true;
+  });
+
+  it('will not recommend showing an error when disabled', async () => {
+    expect(el.validity.valid).to.be.false;
+    el.dispatchEvent(new Event('focus'));
+    await elementUpdated(el as unknown as Element);
+    expect(el.showError).to.be.false;
+    el.dispatchEvent(new Event('blur'));
+    el.toggleAttribute('disabled', true);
+    await elementUpdated(el as unknown as Element);
+    expect(el.showError).to.be.false;
+  });
+
+  it('will get save the validationMessage', async () => {
+    expect(el.validationMessage).to.equal(requiredValidator.message);
+  });
+
+  it('will evaluate when the value is set', async () => {
+    expect(el.validity.valid).to.be.false;
+    expect(el.validity.valueMissing).to.be.true;
+    el.value = 'abc';
+    expect(el.validity.valid).to.be.true;
+    expect(el.validity.valueMissing).to.be.false;
+  });
+
+  it('will respond to validator attribute changes', async () => {
+    const callbackSpy = spy(requiredValidator, 'callback');
+    el.toggleAttribute('required', true);
+    await elementUpdated(el as unknown as Element);
+    expect(callbackSpy.called).to.be.true;
+  });
+});
+
+describe('FormControl checked behavior', () => {
+  let form;
+  let el;
+
+  beforeEach(async () => {
+    form = await fixture(html`<form @submit="${e => e.preventDefault()}">
+      <with-checked name="with-checked"></with-checked>
+    </form>`);
+    el = form.querySelector('with-checked');
+  });
+
+  afterEach(fixtureCleanup);
+
+  it('will not participate in the form if checked is false', async () => {
+    expect(new FormData(form).get('with-checked')).to.equal(null);
```

Comment:
question: is this native behavior? at first glance it seems like there ought to be a form entry just set to false?